src/prelude.rs

```diff
@@ -1,12 +1,15 @@
+#[doc(no_inline)]
 pub use crate::{
+    config::{IronOxideConfig, PolicyCachingConfig},
     document::DocumentOps,
     group::GroupOps,
     internal::{
         document_api::{DocumentId, DocumentName},
         group_api::{GroupId, GroupName},
         user_api::{DeviceId, DeviceName, UserId},
+        DeviceContext, DeviceSigningKeyPair, IronOxideErr, PrivateKey,
     },
     policy::PolicyGrant,
```

Comment:
I'm not sure why this is here, but removing it would make this a breaking change